src/AlgebraicGeometry/Schemes/Sheaves/Types.jl

```diff
@@ -44,19 +44,15 @@ A basic minimal implementation of the interface for `AbsPreSheaf`; to be used in
 
   # production functions for new objects
   is_open_func::Function # To check whether one set is open in the other
-  production_func::Union{Function, Nothing} # To produce ℱ(U) for U ⊂ X
-  restriction_func::Union{Function, Nothing} # To produce the restriction maps ℱ(U) → ℱ(V) for V ⊂ U ⊂ X open
```

Comment:
here again: please check whether the comment still reflects the current situation and change, if necessary. We do not want to confuse programmers who take a look at this as a model.